research-software-directory/RSD-as-a-service
# 118 related software edit

# Related software section

Closes #118 

Changes proposed in this pull request:
*  User is able to edit related software section of the edit software page.    

How to test:
* `docker-compose down --volumes`: remove old database, we did change the structure and row level security
* `docker-compose build`: to build the solution
* `docker-compose up`: to strart app on localhost
* optionally, run data migration script
* add new software
* define related software, on edit page you should be able to link to other RSD software pages, as in the example below

![image](https://user-images.githubusercontent.com/9204081/154981970-c138c5b0-f225-430c-988a-15f21ca97cdd.png)

PR Checklist:
